code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -826,7 +826,10 @@
 	cut_overlays()
 
 	if(stat == CONSCIOUS)
-		add_overlay("eyes-[module_sprites[icontype]]")
+		if(a_intent == "help")
```

Comment:
```suggestion
		if(a_intent == I_HELP)
```

Reply:
I was amazed the first one worked (but it did) and kinda had my suspicions there was a better way to check for intent, I'll see if anything changed mechanically or if it was just a code changed laterish.

Reply:
I've tested it just to see if there were any differences and it works p. much identical

Reply:
`I_HELP` is just a define that means "help", so it will always work identically.